devbox.go

```diff
@@ -23,7 +23,7 @@ type Devbox interface {
 	Install(ctx context.Context) error
 	IsEnvEnabled() bool
 	ListScripts() []string
-	NixEnv(ctx context.Context, opts devopt.NixEnvOpts) (string, error)
+	DevboxEnvExports(ctx context.Context, opts devopt.DevboxEnvExports) (string, error)
```

Comment:
`EnvExports` ? So if the struct name is devbox:

`devbox.EnvExports` ?

Also a bit repetitive but usually our option structs end in `Opt`. We should leave for consistency. (so `EnvExportOpts`)

Reply:
Yeah, lets call it `EnvExports`. Good call.

Yeah, I drop the `Opt` because `devopt` already specifies it. For consistency, I can drop it from the other places as well? Happy to do a follow up.

Reply:
Upon further inspection, I am renaming to `EnvExportOpts` and adding this naming guideline in the devopts package:
```
// Naming Convention:
// - suffix Opts for structs corresponding to a Devbox api function
// - omit suffix Opts for other structs that are composed into an Opts struct
```